scripts/run-code-coverage.sh

```diff
@@ -0,0 +1,21 @@
+# We can't run emulator as a daemon
```

Comment:
It is really bad that VSTS doesn't have an option for emulator. It was in Android Build task but it is deprecated. 😞